test/e2e/config/ironic.yaml

```diff
@@ -47,8 +61,8 @@ intervals:
   default/wait-namespace-deleted: ["10m", "1s"]
   ironic/wait-deployment: ["10m", "2s"]
   default/wait-power-state: ["10m", "100ms"]
-  default/wait-provisioning: ["20s", "1s"]
-  default/wait-provisioned: ["5m", "1s"]
+  default/wait-provisioning: ["1m", "1s"]
+  default/wait-provisioned: ["10m", "1s"]
```

Comment:
Does it take more then 5 minutes to get into provisioned state sometimes? My tests has been around 2-3 minutes.

Reply:
Well. In my trials things get a bit slower in the second cluster. Initially I used a separate wait time for the upgrade tests, but decided to just update the default waits after discussing with Lennart.
I'd say 10m maybe a bit overkill, but 5m would be too short.